modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/rel/set/IgniteColocatedIntersect.java

```diff
@@ -30,6 +30,14 @@
  * Physical node for INTERSECT operator which inputs are colocated.
  */
 public class IgniteColocatedIntersect extends IgniteIntersect implements IgniteColocatedSetOp {
+    /**
+     * Constructor.
+     *
+     * @param cluster   Cluster that this relational expression belongs to.
+     * @param traitSet    The traits of this rel.
```

Comment:
you've got quite peculiar spacing between param name and its description. Please fix it everywhere in the patch

Reply:
Fixed.